AudioKit/Common/MIDI/AKMIDI+Sending.swift

```diff
@@ -230,24 +223,11 @@ extension AKMIDI {
     ///
     public func closeOutput(uid outputUid: MIDIUniqueID) {
         let name = destinationName(for: outputUid)
-        AKLog("Closing MIDI Input '\(String(describing: inputName))'")
+        AKLog("Closing MIDI Output '\(String(describing: name))'")
         var result = noErr
         if let endpoint = endpoints[outputUid] {
-            result = MIDIPortDisconnectSource(outputPort, endpoint)
```

Comment:
'Source' to me sounds like it would be an input, so that makes sense if this is throwing the 'wrong type' error. Can you find the correct function to use (if there is one) to close the output port properly? @karnlund what do you think?